ts/nni_manager/training_service/reusable/trialDispatcher.ts

```diff
@@ -677,14 +682,8 @@ class TrialDispatcher implements TrainingService {
         const environment = environmentService.createEnvironmentInformation(envId, envName);
         environment.environmentService = environmentService;
         this.log.info(`Assign environment service ${environmentService.getName} to environment ${envId}`);
-        environment.command = `sh ../install_nni.sh && python3 -m nni.tools.trial_tool.trial_runner`;
-
-        if (this.isDeveloping) {
-            environment.command = "[ -d \"nni_trial_tool\" ] && echo \"nni_trial_tool exists already\" || (mkdir ./nni_trial_tool && tar -xof ../nni_trial_tool.tar.gz -C ./nni_trial_tool) && pip3 install websockets && " + environment.command;
-        }
-
-        environment.command = `mkdir -p envs/${envId} && cd envs/${envId} && ${environment.command}`;
-
+        environment.command = new LinuxCommands().reusableStartCommand(envId, this.isDeveloping);
+        environment.commandforwin = new WindowsCommands().reusableStartCommand(envId, this.isDeveloping);
```

Comment:
This `commandforwin` seems only for remote windows mode in this PR, don't put it here, this trialDispatcher is for common commands. Suggest to refer localEnvironmentService to see how it process local windows commands, or refactor logic to extract windows command and put it here.

Reply:
Fixed.